platform-common-date/src/main/java/com/softicar/platform/common/date/IllegalDateSpecificationException.java

```diff
@@ -0,0 +1,32 @@
+package com.softicar.platform.common.date;
+
+import com.softicar.platform.common.core.exceptions.SofticarUserException;
+import com.softicar.platform.common.string.Padding;
+
+/**
+ * This {@link Exception} is thrown if an illegal date was specified.
```

Comment:
```suggestion
 * Thrown if an illegal date was specified.
```
...because it's obviously an exception - no need to repeat that.

Reply:
ok